src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static addAnimal(animal) {
+    Animal.alive.push(animal);
+  }
```

Comment:
Is it necessary to create a metod? I'm not sure, but I think, you can just push inside constructor...

Reply:
I think you can do both. Using method allows you to automate it.  So, all animals will be added despite of their type. Just a way to avoid duplication)

Reply:
This method is redundant here